submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/request/SubmissionRequestLoader.kt

```diff
@@ -98,4 +100,16 @@ class SubmissionRequestLoader(
             type = ExtFileType.DIR
         )
     }
+
+    private suspend fun assembleLoadedSubmission(sub: ExtSubmission): ExtSubmission {
```

Comment:
this is super slow as for each file we are hitting the database. Why we are doing this?